### DISCUSSION TITLE
Remove/replace "Air quality" in notification

### DISCUSSION THREAD

**Discussion by SpitFire-666** [breezy-weather/breezy-weather#1535] [Ideas]
### Describe your idea

I don't care about air quality 😀. Could there be an option to remove it or replace it with something I do care about, like min/max temps for the day's forecast? Cheers

![image](https://github.com/user-attachments/assets/6ccdcf4c-8a61-485c-95b3-79d7f06270f5)

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.3.1 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Starting from v5.4.0, you will always be able to select "None" as your air quality, pollen, nowcasting, alerts or normals source, to avoid being overwhelmed by features you don't need.

When air quality is unavailable, the notification-widget in Native mode displays wind data instead.
